app/scripts/models/auth_brokers/fx-fennec-v1.js

```diff
@@ -18,8 +18,7 @@ define(function (require, exports, module) {
 
   var FxFennecV1AuthenticationBroker = FxSyncWebChannelAuthenticationBroker.extend({
     defaultBehaviors: _.extend({}, proto.defaultBehaviors, {
-      afterSignIn: new NavigateBehavior('signin_confirmed'),
-      afterSignUpConfirmationPoll: new NavigateBehavior('signup_confirmed')
```

Comment:
`afterSignUpConfirmationPoll` duplicated a behavior defined in base.js, so I removed it.